src/gui/EQ/EQDrawView.cpp

```diff
@@ -17,6 +18,12 @@ EQDrawView::EQDrawView (const chowdsp::SpectrumPlotBase& plotBase)
 
 void EQDrawView::paint (juce::Graphics& g)
 {
+    const float maxGain = 18.0f;
+    gui::eq::drawDbText (g, spectrumPlot, maxGain);
+    gui::eq::drawFrequencyText (g, spectrumPlot, 100, -maxGain);
+    gui::eq::drawFrequencyText (g, spectrumPlot, 1000, -maxGain);
+    gui::eq::drawFrequencyText (g, spectrumPlot, 10000, -maxGain);
```

Comment:
I'm not sure we actually need to draw these in the DrawView, since the text from the plot should still be visible?